week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/framework/dom-creators.ts

```diff
@@ -0,0 +1,16 @@
+const createElement = (tagName = "div", className = "", idName = "") => {
+  const element = document.createElement(tagName);
+  if (!!className.length) {
+    element.className = className;
+  }
+  if (!!idName.length) {
+    element.id = idName;
+  }
+  return element;
+};
+
+export const p = (className = "") => createElement("p", className);
+export const div = (className = "") => createElement("div", className);
+export const divWithId = (idName = "") => createElement("div", "", idName);
```

Comment:
Jeśli chcesz zrobić dodatkowy helper, to może on korzystać z już istniejącej logiki. np:

```typescript
export const divWithId = (id?: string) => {
   const divWithId = div('')
   if(id) {
      divWithId.id = id
   }
   return divWithId
}
```